pkgs/applications/video/mpv/scripts/mpv-youtube-chat.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildLua,
+  fetchFromGitHub,
+  yt-dlp,
+}:
+
+buildLua {
+  pname = "mpv-youtube-chat";
+  version = "unstable-2024-06-08";
+
+  src = fetchFromGitHub {
+    owner = "BanchouBoo";
+    repo = "mpv-youtube-chat";
+    rev = "4b8d6d5d3ace40d467bc0ed75f3af2a1aefce161";
+    hash = "sha256-uZC7iDYqLUuXnqSLke4j6rLoufc/vFTE6Ehnpu//dxY=";
+  };
+
+  scriptName = "youtube-chat";
+  scriptPath = ".";
```

```suggestion
  scriptPath = "youtube-chat";
```

otherwise it errors at runtime `Cannot find main.* for any supported scripting backend in: /nix/store/77kan7dhygd8msbf63kaqfwb7d2axi94-mpv-youtube-chat-unstable-2024-06-08/share/mpv/scripts/.`